pom.xml

```diff
@@ -20,7 +20,7 @@
   <groupId>software.amazon.event.ruler</groupId>
   <artifactId>event-ruler</artifactId>
   <name>Event Ruler</name>
-  <version>1.0.5</version>
+  <version>1.0.6</version>
```

Comment:
I'd be okay with changing to 1.1.0 for this one. In past, Tim and Long did that for other matchers functionality additions.

Reply:
> I'd be okay with changing to 1.1.0 for this one. In past, Tim and Long did that for other matchers functionality additions.

Per semantic versioning rules, should definitely be 1.1.0, you've added a feature.